ang/civicase/case/actions/directives/case-actions.directive.js

```diff
@@ -368,8 +91,8 @@
         // Special actions when viewing deleted cases
         if (cases.length && cases[0].is_deleted) {
           $scope.caseActions = [
-            { action: 'deleteCases(cases, "delete")', title: ts('Delete Permanently') },
-            { action: 'deleteCases(cases, "restore")', title: ts('Restore from Trash') }
+            { action: 'DeleteCases', type: 'delete', title: ts('Delete Permanently') },
+            { action: 'DeleteCases', type: 'restore', title: ts('Restore from Trash') }
```

Comment:
you don't have to do this in this PR, but I believe it would be better to move including any actions to a config file. This directive should only be about listing actions and should not have special treatment. Similar to what extensions should do to include new actions.